lib/schemaUtils.js

```diff
@@ -634,7 +634,7 @@ module.exports = {
     Object.assign(specComponentsAndUtils, concreteUtils.getRequiredData(spec));
 
     for (path in paths) {
-      if (paths.hasOwnProperty(path)) {
+      if (paths.hasOwnProperty(path) && typeof paths[path] === 'object') {
```

Comment:
@aman-v-singh We'll also have to add `paths[path]` condition because `typeof null = object` in JS.

i.e.
`paths.hasOwnProperty(path) && paths[path] && typeof paths[path] === 'object'`